hr_dms_field/static/src/js/base/dms_list_controller.esm.js

```diff
@@ -0,0 +1,13 @@
+/** @odoo-module **/
+
+import {DMSListControllerObject} from "@dms_field/views/dms_list/dms_list_controller.esm";
+import {patch} from "@web/core/utils/patch";
+
+patch(DMSListControllerObject, "hr_dms_field.DMSListControllerObject", {
```

Comment:
Can you clarify with a code comment why is this needed?

Reply:
Comment added.